From 5c26dd183c4c63fbb5342fcb1c530e0189664090 Mon Sep 17 00:00:00 2001
From: WofWca <wofwca@protonmail.com>
Date: Mon, 4 May 2026 00:21:38 +0400
Subject: [PATCH] improvement: adjust random speed CVARs' defaults

The new values result in 300 random speed for 100 damage
and 500 for 200+ damage.

The old values have been tuned mostly around the vanilla value of 250,
to be somewhat close to the original look.
However, what we should use as a reference is the gibs'
eventual _spread_ which gets smaller with player speed,
because the gibs hit a wall before they can fly far apart.
The low spread was especially noticeable with quad damage kills.
---
 code/cgame/cg_cvar.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/code/cgame/cg_cvar.h b/code/cgame/cg_cvar.h
index a068dda92..0422c053a 100644
--- a/code/cgame/cg_cvar.h
+++ b/code/cgame/cg_cvar.h
@@ -52,10 +52,10 @@ CG_CVAR( cg_oldGibs, "cg_oldGibs", "0", CVAR_ARCHIVE,
 	"When set to 1, takes priority "
 	"over (almost) all the other `cg_gibs*` vars." )
 CG_CVAR( cg_gibsInheritPlayerVelocity, "cg_gibsInheritPlayerVelocity", "1.0", CVAR_ARCHIVE, NULL )
-CG_CVAR( cg_gibsRandomVelocityFromKnockback, "cg_gibsRandomVelocityFromKnockback", "0.15", CVAR_ARCHIVE,
+CG_CVAR( cg_gibsRandomVelocityFromKnockback, "cg_gibsRandomVelocityFromKnockback", "0.4", CVAR_ARCHIVE,
 	"Scale how much random velocity (spread) the gibs gain "
 	"from damage (knockback)." )
-CG_CVAR( cg_gibsExtraRandomVelocity, "cg_gibsExtraRandomVelocity", "175", CVAR_ARCHIVE,
+CG_CVAR( cg_gibsExtraRandomVelocity, "cg_gibsExtraRandomVelocity", "100", CVAR_ARCHIVE,
 	"Like `cg_gibsRandomVelocityFromKnockback`, "
 	"but add a *fixed* amount of random velocity to each piece.\n"
 	"Ensures that even if the player was gibbed with the machine gun, "
-- 
2.55.0

